project/npda/general_functions/csv/csv_download.py

```diff
@@ -41,3 +45,51 @@ def download_xlsx(request, submission_id):
     response = HttpResponse(xlsx_file, content_type="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
     response["Content-Disposition"] = f'attachment; filename="{xlsx_file_name}"'
     return response
+
+
+def export_as_csv(request, submission):
+    out = StringIO()
+    writer = csv.writer(out, delimiter=",")
+
+    pz_code = submission.paediatric_diabetes_unit.pz_code
+    is_jersey = pz_code == "PZ248"
+
+    if is_jersey:
+        HEADINGS_LIST = UNIQUE_IDENTIFIER_JERSEY + CSV_HEADING_OBJECTS
+    else:
+        HEADINGS_LIST = UNIQUE_IDENTIFIER_ENGLAND + CSV_HEADING_OBJECTS
+
+    header = [row["heading"] for row in HEADINGS_LIST]
+    writer.writerow(header)
+
+    visits = Visit.objects.filter(patient__in=submission.patients.all()).select_related("patient").prefetch_related("patient__paediatric_diabetes_units")
```

Comment:
I think this should only do two queries. We need `prefetch_related` for the `Transfer` model (via the now confusingly named `paediatric_diabetes_units` relation) as the foreign key to patient is defined on `Transfer` - legacy from before we decided to have multiple patients with the same NHS number (across submissions and transferring between units)